en/console-and-shells.rst

```diff
@@ -354,6 +354,19 @@ By default, a ``requested`` extra param is automatically added when a Shell
 is dispatched using ``dispatchShell()``. This ``requested`` parameter prevents
 the CakePHP console welcome message from being displayed on dispatched shells.
 
+Exit Status Codes
+-----------------
+It can sometimes be useful to return exit status codes. Simply use php's ``exit()`` statement.
```

Comment:
Printing an error and halting the script is the sole purpose of `$this->error()`.

I agree that true/false being flipped is odd, but I feel that it matches the mental model people have for functions where the function returns a boolean indicating success. The translation to an exit code is a separate concern. Also because we can only safely interpret booleans there is no way to support a variety of exit codes. This is one of the reasons that `_stop()` exists. I could also see use cases where `$this->error()` accepts a status code if that would make more sense to you.